packages/cli/lib/utils.ts

```diff
@@ -176,6 +177,8 @@ export function getFieldType(rawField: string | NangoModel): string {
             case 'date':
                 tsType = 'Date';
                 break;
+            default:
+                tsType = field;
```

Comment:
This will allow us to support `Record<string, string>` and I suppose `Array<User>` as well